components/AlertNotification/AlertNotification.test.tsx

```diff
@@ -1,8 +1,9 @@
 import React from 'react';
-import { render } from '@testing-library/react';
+import { render, screen, fireEvent} from '@testing-library/react';
 import Alert from './AlertNotification';
 import { AlertVariants } from './Alerts.enum';
 import { CheckCircle, XCircle, Info, AlertTriangle } from 'lucide-react';
+import toast from 'react-hot-toast';
```

Comment:
nit: please sort these alphabetically.

I'm aware these weren't sorted alphabetically prior to your work in the file, but we try and do so for ease of scanning the imports.

Reply:
@ryandotfurrer done